draft-ietf-quic-transport.md

```diff
@@ -3227,9 +3208,9 @@ Packet numbers are divided into 3 spaces in QUIC:
 
 - Initial space: All Initial packets ({{packet-initial}}) are in this space.
 - Handshake space: All Handshake packets ({{packet-handshake}}) are in this
-space.
-- Application data space: All 0-RTT and 1-RTT encrypted packets
-  ({{packet-protected}}) are in this space.
```

Comment:
packet-protected is in a different section from packet-initial and packet-handshake, so that reference looks odd in the output.